### DISCUSSION TITLE
cant get groups from openLDAP

### DISCUSSION THREAD

**Discussion by mahdibahramih** [dexidp/dex#2377] [Q&A] [answered]
we need to provide different access levels for k8s dashboard for different teams in our company, so we decide to use dex. we have setup dex in our k8s cluster and connect it to our openLDAP server. everything is good I can get id-token with example-app and access to k8s dashboard with that but we have one problem, we cant get groups member from our LDAP server. my dex config files like bellow:
![Screenshot_20220119_120314](https://user-images.githubusercontent.com/29194581/150093826-8b7bfc73-ce50-4afa-999a-1ccc0cc05bc6.png)

and in dex container logs I can see dex give all info about user from ldap but group array is empty :
"level=info msg="login successful: connector \"ldap\", username=\"Mahdi Bahrami Hidaji\", preferred_username=\"\", email=\"mahdi.bahramihidaji@sexample.com\", groups=[]""

also my group structure in ldap like bellow:
dn: cn=testgroup,ou=Groups,dc=example,dc=local
gidNumber: 532
cn: Tech-Supervisor
objectClass: posixGroup
objectClass: top
memberUid: person1
memberUid: person2
memberUid: person3

**Comment by sagikazarmark** [comment]
`ou=Administrators` seems to be missing from your structure.

**Reply by mahdibahramih** [reply]
@sagikazarmark  thank for your replay.
no "ou=Administrators,ou=Groups,dc=example,dc=local" is exist.

**Comment by mahdibahramih** [comment, marked as answer]
I fixed it.
this solution is worked for me: 
https://github.com/dexidp/dex/issues/744
